### DISCUSSION TITLE
Arrow keys for viewing articles

### DISCUSSION THREAD

**Discussion by adm2k** [FreshRSS/FreshRSS#6442] [Q&A] [answered]
I may be missing something in configuration but, should you not be able to scroll up and down when viewing articles instead of having do a mouse click on the next article?

Thanks...

**Comment by math-GH** [comment]
I am not sure if I understood you correct.

Have you tried this arrow buttons that helps you to navigate through the articles?
![grafik](https://github.com/FreshRSS/FreshRSS/assets/1645099/e5c8bf77-ef38-4b67-a065-84b61b6e7043)

If you haven't available this buttons check you settings ("display" section)
![grafik](https://github.com/FreshRSS/FreshRSS/assets/1645099/8be268c6-11a4-4f12-8b1c-49e90b02a873)


If it does not solve your wish then please describe your need. Screenshots/screencasts could help to understand your point.

**Comment by adm2k** [comment]
@math-GH 

The navigation buttoned work fine but still require a mouse click. I would like just to be able to use the up and down arrows on the keypad for navigation.

**Comment by math-GH** [comment]
Have a look into the shortcut list :)

![grafik](https://github.com/FreshRSS/FreshRSS/assets/1645099/9808d7f8-9984-4ad7-af32-76389624f333)

**Comment by adm2k** [comment]
@math-GH 

Does not work, even tried changing the key. Just want to be able navigate without a mouse click up or down.
![image](https://github.com/FreshRSS/FreshRSS/assets/120423301/e8857479-235b-42de-b147-037961b1126f)

**Reply by math-GH** [reply]
Which keys do you press? What is your browser environment? Which FreshRSS version do you use? Can you navigate via keyboard on demo.freshrss.org?

**Comment by adm2k** [comment]
@math-GH 

On the demo site I have to click on the navigation arrows and that works no key stroke combination seems to work:
Shift i
Shift j

Fresh RSS version 123.1
Using latest version of Chrome

**Reply by math-GH** [reply, marked as answer]
try `j` (next article) and `k` (previous article) (default shortcuts)

**Reply by math-GH** [reply]
Reset to the default shortcuts:
![grafik](https://github.com/FreshRSS/FreshRSS/assets/1645099/8ed45072-8df9-4261-b8e1-177cf6589975)

**Comment by adm2k** [comment]
@math-GH  

Good grief I feel like an idiot! That works fine. Sorry to be such a bother and thanks for your patience!

**Reply by math-GH** [reply]
you are welcome :)
